__e2e__/verifySideNav.e2e.js

```diff
@@ -43,41 +43,56 @@ test.serial('should login successfully', async t => {
   )
   // Click on login btn
   await app.client.click('#loginButton')
-  // // Check that the default network is MainNet
-  await app.client
-    .getValue('#current-network')
-    .catch(error =>
-      t.is(
-        error.message,
-        'An element could not be located on the page using the given search parameters ("#current-network").',
-      ),
-    )
+  await app.client.waitUntilTextExists('//span', 'Manage Wallets')
+  e.pass()
+})
+
+test.serial('Activity page clicked', async e => {
   // click activity tab
   await app.client.click('#history')
-  await app.client.waitUntilWindowLoaded(10000)
-
+  await app.client.waitUntilTextExists('//h3', 'All Activity')
+  e.pass()
+})
+test.serial('Send tab clicked', async e => {
   // click send tab
   await app.client.click('#send')
-  await app.client.waitUntilWindowLoaded(10000)
+  await app.client.waitUntilTextExists('//h3', 'Send Assets')
+  e.pass()
+})
 
+test.serial('Receive tab clicked', async e => {
   // click receive tab
   await app.client.click('#receive')
-  await app.client.waitUntilWindowLoaded(10000)
-
+  await app.client.waitUntilTextExists('//h3', 'Receive Assets')
+  e.pass()
+})
+test.serial('Contacts tab clicked', async e => {
   // click contacts tab
   await app.client.click('#contacts')
-  await app.client.waitUntilWindowLoaded(10000)
-
+  await app.client.waitUntilTextExists('//h3', 'Manage Contacts')
+  e.pass()
+})
+test.serial('Token sale tab clicked', async e => {
   // click token sale tab
   await app.client.click('#tokensale')
-  await app.client.waitUntilWindowLoaded(10000)
-
+  await app.client.waitUntilTextExists('//h3', 'Token Sale')
+  e.pass()
+})
+test.serial('News tab clicked', async e => {
+  // click token sale tab
+  await app.client.click('#News')
+  await app.client.waitUntilTextExists('//h3', 'News')
+  e.pass()
+})
+test.serial('Settings tab clicked', async e => {
   // click settings tab
   await app.client.click('#settings')
-  await app.client.waitUntilWindowLoaded(10000)
-
+  await app.client.waitUntilTextExists('//h3', 'Settings')
+  e.pass()
+})
+test.serial('Logout button clicked', async e => {
   //click logout
-  //await app.client.click('#logout')
-  //await app.client.waitUntilWindowLoaded(10000)
+  await app.client.click('#logout')
+  await app.client.waitUntilTextExists('//div[contains(@class,\'_3M2L1lj7oJRu_0KBN3_v87\')]', 'Login')
```

Comment:
I am worried about checks on dynamically generated class name hashes like this one... Can we just add an `id` tag to the element you are checking instead?